release/_promotion.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+source _publishing.sh
+
 function prepare_jars_for_promotion {
 	trap 'return ${LIFERAY_COMMON_EXIT_CODE_BAD}' ERR
 
-	if [ -z "${LIFERAY_RELEASE_NEXUS_REPOSITORY_PASSWORD}" ] || [ -z "${LIFERAY_RELEASE_NEXUS_REPOSITORY_USER}" ]
+	if [[ -z "${RELEASE_NEXUS_REPOSITORY_USER}" || -z "${RELEASE_NEXUS_REPOSITORY_PASSWORD}" ]] && [ -n "${nexus_repository_name}" ]
```

Comment:
@lucasmiranda0, please check if this is the existing pattern being used in the actual code :wink: